arch/riscv/riscv_features.c

```diff
@@ -1,15 +1,46 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <stdbool.h>
```

Comment:
This is the first use of `stdbool.h` in this project.

Reply:
Thanks for your review. Fixed.